### PR TITLE
Removes Dementia Doctrine, moves Sigil of Submission to Scripts tier

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: Removed Dementia Doctrine, replacing it with an Application sigil.
rscadd: Adds the Sigil of Accession, the previously-mentioned Application sigil, which is much like a Sigil of Submission, except it isn't removed on converting un-mindshielded targets and will disappear after converting a mindshielded target.
tweak: Sigil of Submission is now a Script instead of a Driver; It's unlocked one tier up, so you have to rely on the Drivers you have until you get Scripts, instead of spamming both Driver sigils for free converts.
rscadd: To replace Sigil of Submission in the Driver tier; Added Taunting Tirade, which is a chanted scripture with a very fast invocation, which, on chanting, confuses, dizzies, and briefly stuns nearby non-servants and allows the invoker a brief time to relocate before continuing the chant.
/:cl:

Fixes having too few components interrupting chanting.

### CODE REVIEW

**Comment by Jazaen** [comment]
Hm, so if sec mass-buys implants and implants EVERYONE, then clockcult is kill?

**Comment by coiax** [comment]
Pretty big nerf. I should point out that you can only really afford to spam conversion traps when you have a good component income.

Removal of deimplanting just means implanted people will get killed instead, which isn't the most fun. Maybe make it Application tier?

**Comment by ChangelingRain** [comment]
That's true of most conversion modes, really.

I guess I could move it to Application.

**Comment by Jazaen** [comment]
Would be for the best, since, again, having conversion gamemodes be ended once sec occupies cargo is not fun.

**Comment by coiax** [comment]
Just make a new application tier sigil that takes a bunch of time to work and converts through an implant (removing it in the process). Make it expensive, so only a high tier cult will consider it. I mean, the alternative is giving Ratvar cultists the ability to summon dank surgery tools.

**Comment by InsaneHyena** [comment]
> Ratvar cultists must now take out secguards out of the round.

Wew. Steelpoint is going to be so happy about this.

**Comment by ChangelingRain** [comment]
Converting implanted targets now requires a Sigil of Accession, which is effectively a beefed-up Sigil of Submission that remains after converting non-implanted targets, but is deleted after converting a mindshielded target.

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -383,6 +383,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	var/starttime = world.time
 	progbar = new(invoker, flee_time, invoker)
 	progbar.bar.color = "#AF0AAF"
+	animate(progbar.bar, color = initial(progbar.bar.color), time = flee_time+grace_period)
```
Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: uh

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: yes?

**Pull request merged**